src/lumigo_tracer/lumigo_utils.py

```diff
@@ -34,6 +35,7 @@
 HTTPS_PREFIX = "https://"
 LOG_FORMAT = "#LUMIGO# - %(asctime)s - %(levelname)s - %(message)s"
 SECONDS_TO_TIMEOUT = 0.5
+COOLDOWN_AFTER_TIMEOUT_DURATION = 10
```

Comment:
IMO this is easier to understand
```suggestion
COOLDOWN_AFTER_TIMEOUT_DURATION = datetime.timedelta(seconds=10)
```